test/Microsoft.AspNetCore.Razor.Evolution.Test/TestRazorSourceDocument.cs

```diff
@@ -14,25 +16,34 @@ private TestRazorSourceDocument(string content, Encoding encoding, string filena
         {
         }
 
-        public static RazorSourceDocument CreateResource(string path, Encoding encoding = null)
+        public static RazorSourceDocument CreateResource(string path, Encoding encoding = null, bool normalizeNewLines = false)
```

Comment:
Should this just be the default when you're reading a resource? We want **ALL** of our integration tests to do this unless otherwise specified.